### PR TITLE
Document that my profile is now in settings

### PR DESCRIPTION
**Pull request by ann0see**
Due to @jp8 's PR My Profile is now no longer in "View"

# Does this need translation?

- [x] Yes 
- [ ] No 

# Changes

### CODE REVIEW

**Comment by ann0see** [comment]
@ignotus666 @gilgongo this is kind of urgent...

**Review by ignotus666 — APPROVED** [review]
Wasn't aware of that, thanks for correcting it!

Edit: hold on... I'm not in front of my PC and won't be until tomorrow, but I'm pretty sure the settings button at the bottom left isn't the only path to get to My Profile. I just checked the Getting Started page on the website and some screenshots have more drop-down menus than others at the top... But anyway, one of those also opens the settings window, so I suppose that should be mentioned too, shouldn't it?

**Comment by gilgongo** [comment]
Yes, this is might not be the only change that PR brings to the docs. I'll see if I can check that.

EDIT: OK checked and I think it's just that one screenshot that is out of date (not showing the "settings" item in the menu).

File: wiki/en/Getting-Started.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ When you open Jamulus, you will see a window which looks like this:
 
 ## Setting up a profile
 
-First, let others know who you are. Click on “View” (bar on the top) > “My Profile…” Now you will see this:
+First, let others know who you are. Click on the “Settings” button on the bottom left and navigate to “My Profile…”. Now you will see this:
```
Comment (gilgongo) [inline wiki/en/Getting-Started.md]: I've been trying to use the less specialised "go to" rather than "navigate to" for this.

**Comment by ignotus666** [comment]
> EDIT: OK checked and I think it's just that one screenshot that is out of date (not showing the "settings" item in the menu).

So maybe we could add that another path to My Profile is 'Settings -> My Profile' in the top menu bar (or even Ctrl + p). Or does this complicate things unnecessarily?

File: wiki/en/Getting-Started.md (hunk 1 above)
Comment (ann0see) [reply wiki/en/Getting-Started.md]: Yeah. Just edit it as you‘d prefer.

Comment (gilgongo) [reply wiki/en/Getting-Started.md]:
```suggestion
First, let others know who you are. Click on the “Settings” button on the bottom left and go to “My Profile…”. Now you will see this:
```

**Comment by gilgongo** [comment]
So - shall we go for an "emergency" EN merge to release on this? How does that work for the changes branch?

**Comment by ignotus666** [comment]
> So - shall we go for an "emergency" EN merge to release on this? How does that work for the changes branch?

The change will be automatically synced to next-release so yeah, this can be merged now.

**Pull request merged**